File: src/main/java/com/train/Tickets.java
package com.train;

public class Tickets {

    int TotalTicketCount ;
    int RoundTripTicketCount;

    public Tickets(int totalTicketCount, int roundTripTicketCount) {
        TotalTicketCount = totalTicketCount;
        RoundTripTicketCount = roundTripTicketCount;
    }

    public void conter(){
        int OneRoundTicket=TotalTicketCount-RoundTripTicketCount;
        float Total=OneRoundTicket*1000+(RoundTripTicketCount*2000)*0.9f;
        System.out.println("Toatal tickets:"+TotalTicketCount+"\n"+
                            "Round-trip:"+RoundTripTicketCount+"\n"+
                            "Total:"+Total);
    }
}

File: src/main/kotlin/com/KolinTrain/Tickets.kt
package com.KolinTrain

class Tickets (var TotalTickets:Int,var RoundTripTickets:Int){
    fun counter(){
        var OneRoundTickets:Int=TotalTickets-RoundTripTickets
        var Total:Float = OneRoundTickets*1000+(RoundTripTickets*2000)*0.9f
        println("Total tickets:$TotalTickets \n " +
                "Round-trip tickets:$RoundTripTickets \n"+
                "Total:$Total")

    }
}